app/presenters/summary/sections/base_section.rb

```diff
@@ -29,7 +29,7 @@ def title
         I18n.t(
           name,
           scope: 'summary.sections',
-          subject: I18n.t('summary.dictionary.subject', subject_type:),
+          subject: I18n.t("summary.dictionary.subjects.#{subject_type}"),
```

Comment:
I'm not sure subject_type is working on the outgoings page. The page title knows that it is the client and partner, but the section title doesn't
![image](https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/6758bed9-7e53-4101-933f-b3998810d674)

Reply:
I think we want ownership too, not subject looking at the dictionary